test/msgpack.cpp

```diff
@@ -100,7 +100,8 @@ TEST_CASE(test_msgpack_float)
     migraphx::value v = 3.0;
     auto buffer       = migraphx::to_msgpack(v);
     EXPECT(buffer == msgpack_buffer(3.0));
-    EXPECT(migraphx::from_msgpack(buffer) == v);
+    double epsilon = 1e-9;
+    EXPECT(std::abs(migraphx::from_msgpack(buffer).to<float>() - v.to<float>()) < epsilon);
```

Comment:
Use `test::within_abs` instead.

Reply:
I fixed it in the following [commit](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/2155/commits/bf42d614b24dadc434999797e3404179d1b6a4eb).